var/spack/repos/builtin/packages/openimageio/package.py

```diff
@@ -4,34 +4,28 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 from spack.package import *
-from spack.pkg.builtin.boost import Boost
 
 
 class Openimageio(CMakePackage):
-    """OpenImageIO is a library for reading and writing images, and a bunch of
-    related classes, utilities, and applications."""
+    """Reading, writing, and processing images in a wide variety of file formats, using
+    a format-agnostic API, aimed at VFX applications."""
 
-    homepage = "https://www.openimageio.org"
-    url = "https://github.com/OpenImageIO/oiio/archive/Release-1.8.15.tar.gz"
+    homepage = "https://openimageio.readthedocs.io"
+    git = "https://github.com/AcademySoftwareFoundation/OpenImageIO"
+    url = "https://github.com/AcademySoftwareFoundation/OpenImageIO/archive/refs/tags/v2.5.14.0.tar.gz"
 
     license("Apache-2.0")
 
-    version("2.2.7.0", sha256="857ac83798d6d2bda5d4d11a90618ff19486da2e5a4c4ff022c5976b5746fe8c")
-    version("1.8.15", sha256="4d5b4ed3f2daaed69989f53c0f9364dd87c82dc0a09807b5b6e9008e2426e86f")
+    version("2.5.15.0", sha256="7779ef2c3d03c5ed95e13ff292de85c3f8cee301cd46baad0d2dc83c93bfe85c")
 
-    depends_on("cxx", type="build")  # generated
+    depends_on("cxx", type="build")
 
     # Core dependencies
     depends_on("cmake@3.2.2:", type="build")
-    depends_on("boost@1.53:", type=("build", "link"))
-
-    # TODO: replace this with an explicit list of components of Boost,
-    # for instance depends_on('boost +filesystem')
-    # See https://github.com/spack/spack/pull/22303 for reference
-    depends_on(Boost.with_default_variants, type=("build", "link"))
-    depends_on("libtiff@4.0:", type=("build", "link"))
-    depends_on("openexr@2.3:", type=("build", "link"))
-    depends_on("libpng@1.6:", type=("build", "link"))
+    depends_on("boost+atomic+filesystem+thread+chrono@1.53:")
```

Comment:
@beatreichenbach My changes are in a new commit. You should see my change to add +chrono here.

Reply:
I know you're not gonna believe me but I swear I didn't see the update to 2.5.15.0 and the chrono before haha.
Well thank you for getting that sorted. I see it now 👍